src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -82,6 +83,10 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader->load('fragment_renderer.xml');
 
+        if (class_exists(Application::class)) {
```

Comment:
You should add a `FileExistenceResource` resource so that if the console component is installed, the cache is automatically rebuilt.